package.json

```diff
@@ -1,41 +1,41 @@
 {
-  "name": "kaktana-react-lightweight-charts",
-  "version": "1.1.2",
-  "description": "A simple react wrapper for the tradingview lightweight charts module",
-  "main": "dist/kaktana-react-lightweight-charts.min.js",
-  "types": "types/kaktana-react-lightweight-charts.d.ts",
-  "scripts": {
-    "build": "webpack"
-  },
-  "keywords": [
-    "lightweight",
-    "charts",
-    "react",
-    "wrapper",
-    "tradingview"
-  ],
-  "author": "Kaktana (https://kaktana.com)",
-  "bugs": {
-    "url": "https://github.com/Kaktana/kaktana-react-lightweight-charts/issues"
-  },
-  "homepage": "https://github.com/Kaktana/kaktana-react-lightweight-charts#readme",
-  "license": "MIT",
-  "dependencies": {
-    "fast-deep-equal": "^3.1.1",
-    "lightweight-charts": "^2.0.0"
-  },
-  "peerDependencies": {
-    "react": "^16.13.1"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.9.0",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/preset-env": "^7.9.0",
-    "@babel/preset-react": "^7.9.4",
-    "babel-loader": "^8.1.0",
-    "css-loader": "^3.4.2",
-    "style-loader": "^1.1.3",
-    "webpack": "^4.42.1",
-    "webpack-cli": "^3.3.11"
-  }
-}
+	"name": "kaktana-react-lightweight-charts",
+	"version": "1.1.3",
+	"description": "A simple react wrapper for the tradingview lightweight charts module",
+	"main": "dist/kaktana-react-lightweight-charts.min.js",
+	"types": "types/kaktana-react-lightweight-charts.d.ts",
+	"scripts": {
+		"build": "webpack"
+	},
+	"keywords": [
+		"lightweight",
+		"charts",
+		"react",
+		"wrapper",
+		"tradingview"
+	],
+	"author": "Kaktana (https://kaktana.com)",
+	"bugs": {
+		"url": "https://github.com/Kaktana/kaktana-react-lightweight-charts/issues"
+	},
+	"homepage": "https://github.com/Kaktana/kaktana-react-lightweight-charts#readme",
+	"license": "MIT",
+	"dependencies": {
+		"fast-deep-equal": "^3.1.1",
+		"lightweight-charts": "^2.0.0"
+	},
+	"peerDependencies": {
+		"react": "^16.13.1"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.9.0",
+		"@babel/plugin-proposal-class-properties": "^7.8.3",
+		"@babel/preset-env": "^7.9.0",
+		"@babel/preset-react": "^7.9.4",
+		"babel-loader": "^8.1.0",
+		"css-loader": "^3.4.2",
+		"style-loader": "^1.1.3",
+		"webpack": "^4.42.1",
+		"webpack-cli": "^3.3.11"
+	}
+}
```

Comment:
Please keep the project indentation format of 2 spaces. Also, add a newline at the end of the file

Reply:
yep, i think it was a problem of prettier that formatted the files using the configuration in the .prettierc file.
I've also updated it setting 2 spaces for indentation. let me know if it's ok :)

Reply:
oups my bad, I did a big mistake, it was 4 spaces of indentation indeed. Just the packages.json which was automatically generated didn't respect this so I thought by reading the diff that you made a mistake.

So please revert your last changes and indent with 4 spaces instead of Tabs (which you did in your first commit) mostly on the following lines of your previous commit with 4 tabs instead of 2 ;)
https://github.com/Kaktana/kaktana-react-lightweight-charts/blob/cb775e700cfe116ff98cd1126e2be8ff81ff851f/types/kaktana-react-lightweight-charts.d.ts#L15-L16

Thank you